src/docs/product/accounts/quotas/index.mdx

```diff
@@ -21,46 +21,72 @@ Let’s clarify a few terms to start:
 
-4. **Event repetition**
+### 4. Rate Limits
 
-   Event repetition only applies to error events, not transactions or attachments. The following rules apply for event repetition and your quota:
+If the error event rate limit for a project has been exceeded, and your subscription allows, the event will not be counted. Learn more in [Rate Limiting](/product/accounts/quotas/manage-event-stream-guide/#6-rate-limiting) or in [Error Event Limits](/product/accounts/quotas/#error-event-limits). This does not apply to transactions or attachments. You can add limits for error events and attachments in [sentry.io](https://sentry.io). This does not apply to transaction events.
 
-   - If you have chosen to "Delete & Discard" an issue, then _future_ error events with the same fingerprint do not count toward your quota.
-   - If you previously resolved an issue, a new error event counts toward your quota because it may represent a regression in your code.
-   - If you have chosen to ignore alerts about error events with the same fingerprint, a new event counts toward your quota because the event is still occurring. Learn more in [Inbound Filters](#inbound-data-filters).
+#### Attachment Limits
 
-5. **Spike protection**
-
-   Sentry’s spike protection prevents huge overages in error events from consuming your event capacity. Spike protection is not currently available for transactions. Learn more in [Spike Protection](#spike-protection).
+If you have enabled the storage of crash reports, you may set limits for the maximum number of crash reports that will be stored per issue. To set up these limits, use the slider in the "Store Native Crash Reports" option in your organization's "Security & Privacy" settings.
 
-In addition, depending on your project’s configuration and the plan you subscribe to, Sentry may also check:
+### 6. Event Repetition
 
-6. **Rate limit for the project**
+Event repetition only applies to error events, not transactions or attachments. The following rules apply for event repetition and your quota:
 
-   If the error event rate limit for a project has been exceeded, and your subscription allows, the event will not be counted. Learn more in [Rate Limiting](/product/accounts/quotas/manage-event-stream-guide/#6-rate-limiting) or in [Error Event Limits](/product/accounts/quotas/#error-event-limits). This does not apply to transactions or attachments.
+- If you have chosen to "Delete & Discard" an issue, then _future_ error events with the same fingerprint do not count toward your quota.
+- If you previously resolved an issue, a new error event counts toward your quota because it may represent a regression in your code.
+- If you have chosen to ignore alerts about error events with the same fingerprint, a new event counts toward your quota because the event is still occurring. Learn more in [Inbound Filters](#inbound-data-filters).
```

Comment:
```suggestion
- If you have chosen to ignore alerts about error events with the same fingerprint, a new event counts toward your quota because the event is still occurring. Learn more in the Inbound Filters section of [Manage Your Error Quota](/product/accounts/quotas/manage-event-stream-guide/#6-inbound-data-filters). 
```